crates/sui-types/src/messages.rs

```diff
@@ -834,6 +834,20 @@ impl ProgrammableTransaction {
             })
             .flatten()
     }
+
+    fn move_calls(&self) -> Vec<(&ObjectID, &IdentStr, &IdentStr)> {
```

Comment:
Maybe not something that's changed with this PR, but what about module initializers during publish?

Reply:
This API is I think really only used for events? but agreed that init/upgrade are missing from here. You would need to execute the transaction for that information though